connector/exceptionsconnector/README.md

```diff
@@ -0,0 +1,74 @@
+# Exceptions Connector
+
+| Status                   |               |
+| ------------------------ |---------------|
+| Stability                | [development] |
+| Supported pipeline types | See [Supported Pipeline Types](#supported-pipeline-types)  |
+| Distributions            | [contrib]     |
+
+## Supported Pipeline Types
+
+| [Exporter Pipeline Type] | [Receiver Pipeline Type] |
+| ------------------------ | ------------------------ |
+| traces                   | metrics                  |
+
+Extract metrics from [recorded exceptions](https://opentelemetry.io/docs/reference/specification/trace/semantic_conventions/exceptions/) from spans.
+
+## Overview
+
+Each metric will have _at least_ the following dimensions because they are common across all spans:
+- Service name
+- Span kind
+- Status code
+- Exception message
+- Exception type
+
+This connector lets traces to continue through the pipeline unmodified.
+
+## Configurations
+
+If you are not already familiar with connectors, you may find it helpful to first visit the [Connectors README].
+
+The following settings can be optionally configured:
+- `dimensions`: the list of dimensions to add together with the default dimensions defined above.
+  
+  Each additional dimension is defined with a `name` which is looked up in the span's collection of attributes or resource attributes.
+
+- `dimensions_cache_size`: the max items number of `metric_key_to_dimensions_cache`. If not provided, will use default value size `1000`.
```

Comment:
What happens if the cache size is too small? What do we lose then? Why do we have to use a LRU cache here?